airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/NormalizationActivityImpl.java

```diff
@@ -147,13 +149,16 @@ public NormalizationInput generateNormalizationInput(final StandardSyncInput syn
   private CheckedSupplier<Worker<NormalizationInput, NormalizationSummary>, Exception> getLegacyWorkerFactory(
                                                                                                               final IntegrationLauncherConfig destinationLauncherConfig,
                                                                                                               final JobRunConfig jobRunConfig) {
+    log.error("destination DBT -> {}", destinationLauncherConfig.getSupportDbt());
+    log.error("destination normalization -> {}", destinationLauncherConfig.getNormalizationDockerImage());
```

Comment:
are these error logs intentional?

Reply:
yes, I added them on purpose, but I'd probably rather remove them. They are needed more for development than for the end user